install-devstack-xen.sh

```diff
@@ -4,11 +4,12 @@ set -eu
 function print_usage_and_die
 {
 cat >&2 << EOF
-usage: $0 XENSERVER XENSERVER_PASS PRIVKEY <optional arguments>
+usage: $0 XENSERVER XENSERVER_PASS PRIVKEY NODE_TYPE NODE_NAME [CONTROLLER_IP] <optional arguments>
```

Comment:
yes, if we don't set NODE_TYPE, mark its default value as controller, so we are going to install an all-in-one DevStack env